test/cron_parser_test.dart

```diff
@@ -60,17 +61,9 @@ void main() {
         throwsA(TypeMatcher<AssertionError>()));
   });
 
-  test(
-      'Cron().parse() throws exception if current has been called before first next call',
-      () {
-    var cronIterator = Cron().parse("* * * * *", "Europe/Berlin");
-    expect(
-        () => cronIterator.current(), throwsA(TypeMatcher<AssertionError>()));
-  });
-
   group("Cron().parse() delivers", () {
```

Comment:
Should we split this group into two groups, like `next()` and `previous()`? So then we haven't all tests in one big group
![image](https://user-images.githubusercontent.com/24459435/125301447-4a06f180-e32b-11eb-895b-7f9b94f3c366.png)

Reply:
yes

Reply:
Implemented in: 371dda2d34f1b2de0df4974d2f27529ad46aaeaf

![image](https://user-images.githubusercontent.com/24459435/125325818-ab39bf80-e341-11eb-873e-cbbd1ad5c5d4.png)